ide/templates/ide/project/resource.html

```diff
@@ -78,7 +84,7 @@
             </div>
         </div>
         <div class="control-group well native-only" id="non-font-resource-group">
-            <label class="control-label" for="edit-resource-id">{% trans 'Identifier' %}</label>
+            <label class="control-label" id="edit-resource-id">{% trans 'Identifier' %}</label>
```

Comment:
Oops. Mind you, I guess I was probably half way through trying to fix a label for/id mismatch and then forgot. At the moment a lot of labels in resource.html aren't actually tied to anything, but maybe fixing them all should be done in another PR.